specifications/dsp.profile.md

```diff
@@ -41,7 +41,7 @@ catalog. The `CredentialsSupported` object contains the following properties:
 - `types`: REQUIRED. An array of verifiable credential type strings the credential corresponds to
 - `bindingMethod`: REQUIRED. String that identifies how the credential is bound to the identifier of the
   credential holder.
-- `cryptographicSuites` REQUIRED. An array of strings that identify the cryptographic suites supported for verifying
+- `cryptography` REQUIRED. An array of strings that identify the cryptographic suites supported for verifying
```

Comment:
Perhaps we should replace the term "cryptographic suites" also from the description if the property is renamed.
```suggestion
- `cryptography` REQUIRED. An array of strings that identify the cryptographic standards supported for verifying
```